ext/js/comm/cross-frame-api.js

```diff
@@ -297,14 +300,16 @@ class CrossFrameAPI {
         return this._createCommPort(otherTabId, otherFrameId);
     }
 
-    _createCommPort(otherTabId, otherFrameId) {
-        const details = {
-            name: 'background-cross-frame-communication-port',
-            targetTabId: otherTabId,
-            targetFrameId: otherFrameId
-        };
-        const port = yomichan.connect(null, {name: JSON.stringify(details)});
```

Comment:
I think this is the only usage of `yomichan.connect`, so maybe it can be removed in this change?